Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -695,10 +695,7 @@ extension PaymentSheetTestPlayground {
     // Client-side confirmation handler
     func confirmHandler(_ paymentMethodID: String,
                         _ intentCreationCallback: @escaping (Result<String, Error>) -> Void) {
-        // Client-side confirmation, simulate a delay like we are doing validation on our backend
-        DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
-            intentCreationCallback(.success(self.clientSecret!))
-        }
+        intentCreationCallback(.success(self.clientSecret!))
```

Comment:
nit: I feel like there's a significant difference between calling the callback synchronously vs asynchronously, so we should keep the asyncAfter.  Maybe even put it off the main thread to make it more realistic?  Just don't think waiting 2 seconds buys us anything vs the built-in async delay.

Reply:
Changed this to call on the background thread